src/components/staffing/ProjectCell.js

```diff
@@ -2,23 +2,41 @@ import React from 'react'
 import { Cell } from 'fixed-data-table'
 
 export default class ProjectCell extends React.Component {
-
   static propTypes = {
     data: React.PropTypes.array.isRequired,
     field: React.PropTypes.string,
     rowIndex: React.PropTypes.number,
     onClick: React.PropTypes.func,
-  }
+  };
+  ignoredProjectNames = ['HOLIDAY', 'NO PROJECT NAME', 'Leave UK', 'NOTHERE'];
 
   render() {
     const { rowIndex, field, data, ...props } = this.props
+    const currentWeek = Object.keys(data[rowIndex].staffing).sort()[0]
+    const projects = data[rowIndex].projects
+      ? data[rowIndex].projects.filter(
+          (item) => { return !this.ignoredProjectNames.includes(item) }
+        )
+      : []
```

Comment:
A few comments here:
- you can use the `?.` operator to remove the inline ternary
- you can use the `??` operator to specify a default value when it returns undefined
- you should name the argument of the filter function so that it's clear what it represents (here it should be "project")
- when you create an anonymous function that immediately returns, it's a bit shorter and easier to read to remove the `{}` and the return 


Explanation of the last point:
```js
const fn_1 = (item) => { return !this.ignoredProjectNames.includes(item) }

const fn_2 = (item) => !this.ignoredProjectNames.includes(item)

// fn_1 and fn_2 are strictly equivalent
```

Reply:
For points 1 and 2, eslint complains about an unexpected token, preventing me from making a commit. For the last point, eslint expects a block statement surrounding the arrow body. Fixed point 3.

Reply:
Let's update eslint then :D  

(we might do that in a separate ticket if we want to deliver the feature to Zoe quickly, but we should improve the working conditions ASAP #[Lean 5S](https://www.epa.gov/sustainability/lean-thinking-and-methods-5s) → like in the next ticket! feel free to create it)

Reply:
Discussed this in problem solving this morning and agreed to make a ticket to update the ESLint rules